client/components/repository/common/Sidebar/Relationship.vue

```diff
@@ -131,15 +131,13 @@ export default {
     margin-right: 1rem;
   }
 
-  .v-input__slot .v-select__slot {
-    .v-input__append-inner {
-      margin-top: 1.375rem;
-    }
+  .v-input__slot .v-select__slot input[disabled] {
+    opacity: 0.7;
+    border-bottom: unset;
+  }
 
-    input[disabled] {
-      opacity: 0.7;
-      border-bottom: unset;
-    }
+  .v-select.v-select--chips:not(.v-text-field--single).v-text-field--enclosed .v-select__selections {
```

Comment:
The reason for this level of specificity lies in the fact that Vuetify introduces its own `margin-top` value which causes the issue. Without it, Vuetify's declaration would override any attempt at setting `min-height`.

Reply:
Consider using [v-autocomplete](https://vuetifyjs.com/en/api/v-autocomplete/#links) height prop 😉

Reply:
Addressed in 801295fc4ac3d1d18d6a76973b14329b072ebe9f.